terraform/terraform.tfvars.example

```diff
@@ -1,6 +1,6 @@
-project = "clgcporg2-067"
+project = "clgcporg2-092"
 public_key_path = "~/github_repo/devops_study_infra/packer/scripts/TXT.pub"
 disk_image = "reddit-full"
 private_key_path = "~/.ssh/appuser"
 count_vm = 2
-source_image = "projects/clgcporg2-067/global/images/reddit-full-1665389936"
+source_image = "projects/clgcporg2-092/global/images/reddit-full-1665389936"
```

Comment:
here is an example file so you must not change any "example" value to the real existing